tools/e2etesting/TestEventProcessor/TestEventProcessor.Businesslogic/TelemetryValidator.cs

```diff
@@ -262,7 +262,7 @@ private Task CheckForMissingValueChangesAsync(CancellationToken token)
                                     iotHubEnqueuedTime.ToString(_dateTimeFormat, formatInfoProvider),
                                     durationDifference);
                             }
-                            if (Math.Round(durationDifference.TotalMilliseconds) > _currentConfiguration.ExpectedMaximalDuration)
+                            if (Math.Round(durationDifference.TotalMilliseconds) > _currentConfiguration.ExpectedMaximalDuration) 
```

Comment:
[Trailing Whitespace Visualizer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer) is a nice Visual Studio extension that visualizes trailing whitespaces (like here) and removes them when you save the file. Please have a look.

Reply:
Thanks, I installed it now. :-)